### ISSUE TITLE
Error `join_all` not found when adding ractor crate in windows environment.

### ISSUE THREAD

**Issue by mystique09** [slawlor/ractor#290]
**Describe the bug**
Error when adding ractor crate into a project, not sure if this is a tokio-related issue because the error only occurs when adding the ractor crate.

**To Reproduce**
1. Open any rust project.
2. Open `Cargo.toml` file
3. Add ractor crate
4. See error: 
```
no method named `join_all` found for struct `tokio::task::JoinSet` in the current scope
method not found in `JoinSet<Result<(), RactorErr<StopMessage>>>`
```

**Expected behavior**
- To be able to add ractor without any errors.

**Additional context**
Add any other context about the problem here (e.g. Rust version, other crates imported, etc)
- Windows
- cargo 1.81.0 (2dbb1af80 2024-08-20)
- rustc 1.81.0 (eeb90cda1 2024-09-04)
- VS CODE: 1.95.3
f1a4fb101478ce6ec82fe9627c43efbf9e98c813
x64

![image](https://github.com/user-attachments/assets/864425c9-ff83-4221-a622-53a5921896fb)

**Comment by slawlor** [comment]
What version of Tokio do you have? This is recent in Tokio and needed an upgraded version. It's been fixed in main for our dep tree but I haven't pushed a new version yet

**Comment by mystique09** [comment]
> What version of Tokio do you have? This is recent in Tokio and needed an upgraded version. It's been fixed in main for our dep tree but I haven't pushed a new version yet

Currently using version 1.38.0 of tokio.

**Comment by slawlor** [comment]
If you are willing, upgrade to the latest version and you should be fine. I'll try and refactor out the need for such a high Tokio version soon

**Comment by slawlor** [comment]
If you are willing, upgrade to the latest version and you should be fine. I'll try and refactor out the need for such a high Tokio version soon

**Comment by slawlor** [comment]
#291 will address this, and once the signals are clear, i'll push a crate update (v0.13.1)

**Comment by slawlor** [comment]
Fixed in release 0.13.1.

**Comment by mystique09** [comment]
Thanks
